shared/js/background/events.js

```diff
@@ -329,6 +329,15 @@ browser.runtime.onMessage.addListener((req, sender) => {
         return Promise.resolve(messageHandlers[req.messageType](req.options, sender, req))
     }
 
+    // Count refreshes per page
+    if (req.pageReloaded && (sender.tab !== undefined)) {
+        const tab = tabManager.get({ tabId: sender.tab.id })
+        if (tab) {
+            tab.userRefreshCount += 1
```

Comment:
When does this count get reset?

Reply:
When the site gets updated: https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/616b44a712d96b7bbc37c84507dae8c1f2c2a10b/shared/js/background/classes/tab.js#L313